addons/sys_rack/CfgEventHandlers.hpp

```diff
@@ -15,3 +15,21 @@ class Extended_PostInit_EventHandlers {
         clientInit = QUOTE(call COMPILE_FILE(XEH_postInit));
     };
 };
+
+class Extended_InitPost_EventHandlers {
```

Comment:
I did not change this since this is what snippers coded. Once all this is ready and working as expected (no warnings in the rpt as the ones we observed) I will put it up to date.